README.md

```diff
@@ -21,7 +21,10 @@ Options:
   --odoo-version TEXT           Project's Odoo version (e.g. 16.0)  [required]
   --project-name TEXT           Name of the project, will be the name of
                                 category of local addons (default: Local)
-  --oca-category                Add category for third party addons coming from OCA
+  --oca-category                Add category for third party addons coming from OCA.
+                                The category is set as 'OCA/repository_name'.
+                                If the repository can not be identified,
+                                it falls into the default 'OCA' category.
```

Comment:
It would be nice if we could have the grouping by OCA repo optional, as I personally prefer to have a single category for OCA.

Reply:
Done, we have 'three levels of categories':
1) not setting '--oca-cateogry': oca addons fall into 'Third-party' category
2) '--oca-cateogry=basic': oca addons are grouped under 'OCA' category
3) '--oca-category=repository': oca addons are grouped by repository 'OCA/<repository_name>'